presto-docs/src/main/sphinx/develop/serialized-page.rst

```diff
@@ -0,0 +1,234 @@
+=================================
+SerializedPage Wire Format
+=================================
+
+Presto uses SerializedPage binary columnar format for exchanging data between
+stages.
+
+The data can be compressed, encrypted and include a checksum. The layout is a
+header followed by a number of columns, followed by individual columns.
+
+.. image:: serialized-page-layout.png
+  :width: 600
+
+Header
+------
+
+The header includes:
+
+=======================   =======
+Field                     Size
+=======================   =======
+Number of rows            4 bytes
```

Comment:
What do you think about mentioning how integers are encoded (e.g.:  signed two's complement integer)?